### DISCUSSION TITLE
Connectors are not visible

### DISCUSSION THREAD

**Discussion by 0xLDev** [xyflow/xyflow#2898] [Q&A]
Help solve the problem, I have a parent node in which there are children, if the parent has a background then the connectors are not visible, but if you remove the background everything is visible, screenshots are applied
![1](https://user-images.githubusercontent.com/67437278/223346662-540667b3-ea6c-4db4-b3d3-5c36170a70f2.jpg)
![2](https://user-images.githubusercontent.com/67437278/223346668-133be221-77a2-4f73-bc23-aef07d555939.jpg)

**Comment by moklick** [comment]
You need to pass a higher zIndex to your edges https://reactflow.dev/docs/api/edges/edge-options/#edge. By default all edges are displayed below the nodes.

**Reply by 0xLDev** [reply]
That way, the lines will hit the blocks themselves when they drag the nodes, and I don’t need it
![загруженное](https://user-images.githubusercontent.com/67437278/223764023-ec2b3759-f670-4f3a-b055-4d00828ea52c.png)
